compose/config/device/fan.yaml

```diff
@@ -0,0 +1,11 @@
+version: 3
+devices:
+  - type: led
```

Comment:
this is an LED device, but the file is named `fan.yaml`

Reply:
this too